### PR TITLE
Update foreman-tasks to 0.7.1

### PR DESCRIPTION
**Pull request by iNecas**
The deb version will be released after
https://github.com/theforeman/foreman_chef/pull/23 is merged

### CODE REVIEW

**Comment by domcleal** [comment]
@iNecas would you or @stbenjam mind making the same version change in foreman_salt?

File: rubygem-foreman-tasks/rubygem-foreman-tasks.spec [hunk 1]
```diff
@@ -34,8 +34,8 @@ Requires: %{?scl_prefix}ruby(release)
 Requires: %{?scl_prefix}ruby(abi)
 %endif
 
-Requires: %{?scl_prefix}rubygem(dynflow) >= 0.7.9
-Requires: %{?scl_prefix}rubygem(dynflow) < 0.8.0
+Requires: %{?scl_prefix}rubygem(dynflow) >= 0.8.0
```
Comment (domcleal) [inline rubygem-foreman-tasks/rubygem-foreman-tasks.spec]: `>= 0.8.1`, ditto below

**Comment by domcleal** [comment]
foreman_salt's done, thanks @stbenjam.  Regarding the deb stuff, please just go ahead with the PRs for dynflow + foreman-tasks if you're willing, it's much easier to work up the stack bit by bit.

File: rubygem-foreman-tasks/rubygem-foreman-tasks.spec (hunk 1 above)
Comment (iNecas) [inline rubygem-foreman-tasks/rubygem-foreman-tasks.spec]: Updated

**Comment by iNecas** [comment]
@domcleal ad the debian: I was not sure how is it, when we have different version of the foreman-tasks than the foreman_chef actually needs… would the multi-versioned variant actually work?

**Comment by domcleal** [comment]
Depends if the package has version deps matching the gems - ruby-foreman-chef does, so it should be fine, apt should correctly resolve it.  ruby-foreman-salt didn't, but that'll be fixed shortly.  It's only nightlies, so little harm if the versions aren't quite right for a day.

**Comment by domcleal** [comment]
Thanks @iNecas!  Building.

**Pull request merged**